From 6c1c1e7d7443cb14b37530d798c9d875e0059060 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Sat, 17 Sep 2022 21:51:52 +0800
Subject: [PATCH 2/2] fdts: s32g: Change the ptp clock name to 'ptp_ref'

The ptp clock name in s32cc.dtsi is 'ptp_ref' and the s32g eth driver
in u-boot also use the name 'ptp_ref' when trying to enable ptp clock.
So there is no reason to use the 'ts' as the ptp clock name in
s32g.dtsi. This fixes the following ptp error in u-boot.
  Failed to get ptp_ref
  Failed to enable ptp_ref clock

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 fdts/s32g.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 9fa29a86cfd0..be36fcadc7db 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -278,7 +278,7 @@
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_RX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_TX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_AXI>;
-	clock-names = "ts", "rx_sgmii", "tx_sgmii",
+	clock-names = "ptp_ref", "rx_sgmii", "tx_sgmii",
 		      "rx_rgmii", "tx_rgmii", "axi";
 	pinctrl-0 = <&pinctrl0_gmac0 &pinctrl0_gmac0_mdio>;
 	pinctrl-1 = <&pinctrl0_gmac0_mdio>;
-- 
2.34.1

